llama_index/llms/openai.py

```diff
@@ -37,6 +37,13 @@
 )
 
 
+class _Tokenizer(Protocol):
```

Comment:
You'll wanna make this public (e.g. `Tokenizer`) so it can be imported into `openai_like`